transcom/mymove
# INT-B-20941-ssw-duplicate-state-zip

## [B-20941](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A997016)

## Summary

**INTEGRATION**

This PR fixes a small issue in the SSW PDF for AOA/Payment download where Authorized Origin field was displaying duplicate state and zip.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Login as customer to create and submit move with PPM. Make sure to click YES for "Would you like to request an advance on your incentive?"
2. Login as a SC and fill in all required fields and click submit move details. Page will refresh containing AOA download link. Download and verify Authorized Origin field does not contain duplicate state/zip.
3. Login as customer and click upload PPM Documents and complete/submit process of providing documents for review. 
4. Login as SC and review documents. Approve/Reject and confirm/submit. Page will refresh containing Payment Download link. Download and verify Authorized Origin field does not contain duplicate state/zip.
5. Login as customer and click Payment Download  and verify Authorized Origin field does not contain duplicate state/zip.

<img width="316" alt="Screenshot 2024-08-22 at 9 31 19 AM" src="https://github.com/user-attachments/assets/0ce37a29-90b4-4340-a37e-27275a6ec43f">


### Frontend


### Backend
